data/FORMES.md

```diff
@@ -192,6 +192,6 @@ Dynamax and Gigantamax are not considered formes by the games themselves. PS imp
 `pokedex.js`
```

Comment:
We should probably change this to `.ts`?

Reply:
There are other references to `.js` files that have since been TypeScripted; should I just make a separate commit to fix it?

Reply:
Probably, yeah.